### ISSUE TITLE
Can not install using EXE or ZIP

### ISSUE THREAD

**Issue by JcRai10** [erfajo/OrchidForDynamo#112]
#### Install/open
After downloading both the EXE and the ZIP files, I can not open them, even the zip file says it is not a zip file.

#### 
all versions all files

**Comment by erfajo** [comment]
I have now downloaded and tested several zip files and exe files, none gave me an error. Could you be precise about exactly which zip/exe file is failing?

**Comment by Watchitgo333** [comment]
Hey erfajo, not sure if this is still an issue, but I was having issues with the 212 installer for Revit 2022. I noticed the folder was being placed in my C:\ drive amongst my program files folder etc, I then moved it to my C:\Users\Eric\AppData\Roaming\Dynamo\Dynamo Revit\2.10\packages, I loaded dynamo and saw Orchid still wasn't showing up. I tried placing the 2.6 version of Orchid in my 2.10 for Dynamo and saw that Orchid was available. Not sure what the reasoning was, do I have an older version of Dynamo for Revit 2022?
